env0/utils.go

```diff
@@ -42,7 +42,13 @@ func readResourceData(i interface{}, d *schema.ResourceData) error {
 	for i := 0; i < val.NumField(); i++ {
 		fieldName := val.Type().Field(i).Name
 		// Assumes golang is CamalCase and Terraform is snake_case.
+		// This behavior can be overrided be used in the 'resource' tag.
 		fieldNameSC := toSnakeCase(fieldName)
+		if resFieldName, ok := val.Type().Field(i).Tag.Lookup("resource"); ok {
```

Comment:
This is for the new tag I created (resource). Ping me if this is still unclear.

Reply:
Very cool 👏 
Just wondering together with you whether `resourcee` is a decent tag name or we can come up with a better name. Maybe something that indicates that this is related to the TF schema would do better.

Also - WDYT about adding some note regarding it in `DEVELOPMENT.md`?

Reply:
yes and yes.
How about tfschema or tfresource ? (I'm bad with names).

Reply:
Both are better. If you want me to choose then I'd go with `tschema` 😅

Reply:
we have a winner (: